templates/aws/airgap_cluster_nodes/manifest.yaml

```diff
@@ -0,0 +1,74 @@
+name: aws-airgap-cluster-nodes
```

Comment:
I am a bit worried that copying the same terraform module multiple times for minor tweaks/changing node pool names may not be what we want. Is there a way we can make this more configurable? Perhaps we can use templates to use the same terraform, but autogenerate the outputs section?

Reply:
Sounds good, thanks Jake. I'll go back and attempt to make this more configurable.